engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -168,6 +168,7 @@ public void runBare() throws Throwable {
             "Data/Text/Extensions.enso",
             "Data/Text/Regex/Match_2.enso",
             "Data/Text/Regex/Pattern_2.enso",
+            "Data/Text/Regex/Replacer.enso",
```

Comment:
This test is being removed https://github.com/enso-org/enso/pull/5917#discussion_r1135034752 - e.g. changes here are completely OK.